aip/0159.md

```diff
@@ -31,10 +31,6 @@ GET /v1/publishers/-/books?filter=...
 - The resources provided in the response **must** use the canonical name of the
   resource, with the actual parrent collection identifiers (instead of `-`).
 
-**Important:** This pattern **must not** be used for location-bound resources
-to search across locations. Retrieving resources across locations adds
-additional failure modes that require special handling.
-
 ### Unique resource lookup
```

Comment:
Is the unique resource lookup only permitted when the "sub-ids" are *guaranteed* to be unique? Suppose I have names that are *unlikely* to clash, but might still:

- Can the API still support cross-collection lookup?
- What should it do if there's a collision?

Prohibiting it is probably the simplest approach, but I think that's worth calling out.